cfme/services/catalogs/service_catalogs.py

```diff
@@ -53,7 +53,7 @@ def order(self):
         if self.stack_data:
             stack_form.fill(self.stack_data)
         sel.click(form_buttons.submit)
-        wait_for(flash.get_messages, num_sec=10, delay=2, fail_condition=[], fail_func=tb.refresh())
+        sel.sleep(5)
```

Comment:
wait_for(flash.get_messages, num_sec=10, delay=2, fail_condition=[], fail_func=tb.refresh()) and 
wait_for(flash.get_messages, num_sec=10, delay=2) are not working and test fails with 
FlashMessageException: No matching info flash message for 'Order Request was Submitted', instead got []

with sel.sleep(5) it worked but  should it be used ?